src/styles/components/Button.scss

```diff
@@ -25,46 +25,141 @@
         border-color: $btn-default;
         size: $btn-size;
         height: $btn-height;
-   
-    btn-link{
+        box-shadow: none !important;
+
+        &:focus{
+            @include button-focus;
+        }
+    
+        &:hover{     
+            border-color: $btn-hover;
+        }
+    
+        &:active{
+            border: $blue-600 1px solid;
+            background-color: $blue-100;
+        }
+    
+        &:disabled{
+            border: $gray-500 $border-default solid;
+            background-color: $gray-200 !important;
+            color: $gray-450 !important;
+            font-size: $font-size-7;
+        }
+
+    &.btn-link{
         @include link_button;
+        border: none;
+        outline-style: none;
+
+        &:focus{
+            @include button-focus;
+        }
+    
+        &:hover{     
+            color: $btn-hover;
+        }
+    
+        &:active{
+            color: $blue-800;
+            background-color: transparent !important;
+        }
+    
+        &:disabled{
+            border: none;
+            background-color: $gray-200 !important;
```

Comment:
Can you please reuse these already declared variables here instead of codes, so it will be consistent for all elements and more readable?
$disabled-color, $disabled-background-color

Reply:
Done.